### DISCUSSION TITLE
Patch 7.1 - API 11 Update

### DISCUSSION THREAD

**Discussion by reiichi001** [goatcorp/Dalamud#2071] [Announcements]
Hi developers! We're bringing you a quick update about Dalamud API 11 scheduled to be released together with Patch 7.1.
At present, the currently planned significant changes are as follows:

We are [upgrading to Lumina 5](https://github.com/goatcorp/dalamud-docs/blob/main/docs/versions/v11/lumina.md), which promises speed and memory improvements across the board.
We are bringing in the [new breaking changes from Client Structs](https://github.com/aers/FFXIVClientStructs/blob/main/7.1_changes.md) to clean up the API surface and better reflect some new findings.
We are targeting a bump to .NET 9, which is due for release this week. This will be delayed if API 11 needs to be released prior to .NET 9's release.

The [api11 branch](https://github.com/goatcorp/Dalamud/tree/api11) of Dalamud has been enabled and is now properly tracking Plugin API version 11, so you should be able to update things accordingly. Please note that the API surface may change slightly between now and Patch 7.11, though we expect most things to be stable. Internal changes are still being made and reviewed.

You may view the changes between API 10 and API 11 in [PR #2049](https://github.com/goatcorp/Dalamud/pull/2049).

### DalamudPackager and SDK
DalamudPackager and Dalamud.NET.Sdk have both been updated to 11.0.0 to support the new API level. Notably, this aligns the versioning scheme for DalamudPackager to that of the SDK and pins it to the major Dalamud version.

**Comment by reiichi001** [comment]
EDIT: Square changed things, and there have been updates to a few notable systems, including sheets. We are now targeting API 11 for Patch 7.1 proper.

**Comment by goaaats** [comment]
DalamudPackager and Dalamud.NET.Sdk have both been updated to 11.0.0 to support the new API level. Notably, this aligns the versioning scheme for DalamudPackager to that of the SDK and pins it to the major Dalamud version.
